web/src/lib/utils/slideshow-history.ts

```diff
@@ -0,0 +1,40 @@
+export class SlideshowHistory {
+  private slideshowHistory: string[] = [];
+  private slideshowHistoryIndex = 0;
```

Comment:
these are inside a class called `Slideshow*`, so you could probably omit `slideshow*` from the variable names.